app/features/send/send-input.tsx

```diff
@@ -187,15 +228,23 @@ export function SendInput() {
         </div>
 
         <div className="flex h-[24px] items-center justify-center gap-4">
-          {paymentRequest && (
+          {(contact || lud16 || paymentRequest) && (
```

Comment:
this seems pretty hard to read. I'd rather have the state have something like sendType and then when you select contact it would set the sendType to that when you select ln address it would set it to that. it would also be updating the state of the send flow so we'd be able to check that instead of bool flags that can go out of sync. would also mean we could get rid of code like this:
```
setLud16(null);
setContact(null);
clearPaymentRequest();
```